tools/reflection_generator/java_method.py

```diff
@@ -482,12 +489,18 @@ def GenerateBridgeConstructor(self):
 
         this.wrapper = wrapper;
         reflectionInit();
+${POST_BRID_LINES}
     }
 
     """)
+
+      post_bird_string = self._method_annotations.get(
```

Comment:
typo, it should be brid.
I just refer to post_wrapline_re, I don't know why it not use post_wrapperline_re.
Do you think post_bridge_string is better?